environment.yml

```diff
@@ -0,0 +1,11 @@
+name: fresnel
+channels:
+  - conda-forge
+dependencies:
+  - fresnel==0.12.0
+  - jupyterlab==2.2.8
```

Comment:
Do we need to pin everything? I'd prefer not to have to constantly update these and leave them unpinned so builds always get the latest versions. Fresnel itself needs to be pinned, of course.

Reply:
I don't like pinning everything, but I thought it would increase the chances that everything stays working. I had some issues with the version of `qhull` that got pulled in, other fragile things can be `ipywidgets` being the correct version for `jupyterlab`. 

Perhaps we can only pin `fresnel` and `qhul` for now until something breaks? I just know if we don't regularly test this environment with the notebooks, it will take a user hitting an error and then being motivated enough to make an issue/communicate the error. Perhaps adding a github action that tests the notebooks weekly using nbval and then environment file would be enough.

Reply:
What issues did you have with qhull? Is it possible that the fresnel conda package needs to be rebuilt to support the latest version?